modules/appeals_api/app/workers/appeals_api/hlr_pdf_submit_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'appeals_api/hlr_pdf_constructor'
+
+module AppealsApi
+  class HlrPdfSubmitJob
```

Comment:
should we use the HLR abbeviation here?

Reply:
No, I will change these. the abbreviation a bad idea.

Reply:
updated